src/ui/common/src/utils/operators.ts

```diff
@@ -135,9 +135,10 @@ export function normalizeOperator(op): Operator {
 }
 
 export type GetOperatorResultResponse = {
-  status: ExecutionStatus;
-  error: string;
-  logs: { [name: string]: string };
+  code: ExecutionStatus;
```

Comment:
Is there a reason we renamed this from `status` to `code`? The word code in my mind is like an error code (e.g., when a process does `exit(1)` or `exit(127)`. We're just using strings here, so status seems fine, but let me know if I'm missing something.

Reply:
Yeah status sounds good :)